site/projects/surface_upper_air.md

```diff
@@ -5,9 +5,11 @@ Declarative Surface/Upper Air Map
 
 ## Description
 
-_description here_
+This project aims to equip students with the skills needed to use MetPy's declarative plotting interface to generate surface and upper air maps. In successfully completing this project, students will gain experience using Siphon to access remote, satellite data from the THREDDS Data Server (TDS). Students will also learn how to process and display meteorological data using Python-AWIPS and Cartopy.
 
 ## Related Training Notebooks
+* [Surface Data with Siphon and MetPy](https://unidata.github.io/python-training/workshop/Surface_Data/surface-data-with-siphon-and-metpy/)
```

Comment:
As with the satellite page, I'd recommend adding a "Related Comprehensive Notebooks" subsection to hold these link.